cfg/default.json

```diff
@@ -25,6 +25,7 @@
         wide_trans: 32,
         dma_user_width: 48,
         enable_multicast: true,
+	enable_dma: true,
```

Comment:
This new parameter should be added to the `util/clustergen/snitch_cluster.schema.json` file. There, you can set the default value to true, so you can remove the changes from all json files but the ones you created.